packages/sg2/src/msg.rs

```diff
@@ -30,6 +30,9 @@ pub struct UpdateMinterParamsMsg<T> {
     pub min_mint_price: Option<Coin>,
     pub mint_fee_bps: Option<u64>,
     pub max_trading_offset_secs: Option<u64>,
+    pub max_royalty_bps: Option<u64>,
+    pub max_royalty_increase_rate_bps: Option<u64>,
+    pub royalty_min_time_duration_secs: Option<u64>,
```

Comment:
Instead of passing this in, I think its safe to assume 24 hours.

Reply:
refactored to hardcode 24 hours